config/crd/bases/cluster.x-k8s.io_clusters.yaml

```diff
@@ -23,7 +23,6 @@ spec:
     shortNames:
     - cl
     singular: cluster
-  preserveUnknownFields: false
```

Comment:
We likely don't want to drop this, it may require updating the kubebuilder annotation for Cluster to set `crd:preserveUnknownFields=false` as per https://github.com/kubernetes-sigs/controller-tools/pull/362